src/AppBundle/PullRequests/Listener.php

```diff
@@ -15,6 +18,10 @@ class Listener
     const PRESTONBOT_NAME = 'prestonBot';
     const TABLE_ERROR = 'PR_TABLE_DESCRIPTION_ERROR';
     const COMMIT_ERROR = 'PR_COMMIT_NAME_ERROR';
+    const FEEDBACK_COMMENT = 'PR_FEEDBACK_COMMENT';
+
```

Comment:
```suggestion
```

Reply:
I did it on purpose to make it more readable by separating Twig-related const  and other const.